### PR TITLE
Docs - Block Editor Handbook - Add end user documentation about Block Editor as a resource on the Landing Page of the Block Editor Handbook

### PR DESCRIPTION
**Pull request by juanmaguitar**
## What?
Add end user documentation about Block Editor as a resource on the Landing Page of the Block Editor Handbook

## Why?
To improve the learning path of developers and users

### CODE REVIEW

**Comment by mburridge** [comment]
TBH I'm not sure that we need to add a link to user documentation as an additional resource in what is essentially developer documentation. What is your rationale for doing this? 

@ryanwelcher, @ndiego, @justintadlock what do you think?

**Comment by juanmaguitar** [comment]
@ndiego @ryanwelcher any thoughts on this PR?

File: docs/README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Gutenberg is open source software and anyone is welcome to contribute to the pro
 
 This handbook should be considered the canonical resource for all things related to block development. However there are other resources that can help you.
 
+- [**End User Documentation**](https://wordpress.org/documentation/) - Documentation site targeted to the end user (not developers) where you can also find documentation about the [Block Editor](https://wordpress.org/documentation/category/block-editor/) and the [work with blocks](https://wordpress.org/documentation/article/work-with-blocks/)
```
Comment (ndiego) [inline docs/README.md]:
```suggestion
- [**End User Documentation**](https://wordpress.org/documentation/) - Documentation site targeted to the end user (not developers) where you can also find documentation about the [Block Editor](https://wordpress.org/documentation/category/block-editor/) and [working with blocks](https://wordpress.org/documentation/article/work-with-blocks/).
```

Just a minor grammar tweak, and I might move this to the bottom of the list.

**Review by ndiego — APPROVED** [review]
This seems like a fine resource to add. I did note a non-blocking fix.

**Pull request merged**